keyboards/dztech/dz60rgb/keymaps/mekanist/keymap.c

```diff
@@ -8,38 +8,38 @@
+		KC_NO,    RGB_SAD,  RGB_SAI,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,              KC_NO , \
+		KC_NO,    RGB_VAD,  RGB_VAI,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    RGB_MOD,            KC_TRNS, \
+		KC_NO,    KC_NO,    KC_NO,                        KC_NO,                        KC_NO,    KC_NO,    RGB_SPD,  RGB_RMOD,           RGB_SPI),
 	[_LAYER3] = LAYOUT( /* NUMPAD */
-		KC_NLCK,  KC_PSLS,  KC_PAST,  KC_PMNS,  KC_PPLS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_PSLS,  KC_PAST,  KC_PMNS,  KC_PPLS,  KC_TRNS, \
-		KC_TRNS,  KC_P7,    KC_P8,    KC_P9,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_P7,    KC_P8,    KC_P9,    KC_TRNS,  KC_TRNS,  TO(0), \
-		KC_TRNS,  KC_P4,    KC_P5,    KC_P6,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_P4,    KC_P5,    KC_P6,    KC_TRNS,            KC_PENT, \
-		KC_TRNS,  KC_P1,    KC_P2,    KC_P3,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_P1,    KC_P2,    KC_P3,    KC_TRNS,            KC_TRNS, \
-		KC_TRNS,  KC_P0,    KC_PDOT,                      KC_PENT,                      KC_P0,    KC_PDOT,  KC_TRNS,  KC_TRNS,            KC_TRNS),
-	[_LAYER4] = LAYOUT( /* MAC */
+		KC_NLCK,  KC_PSLS,  KC_PAST,  KC_PMNS,  KC_PPLS,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_PSLS,  KC_PAST,  KC_PMNS,  KC_PPLS,  KC_NO, \
+		KC_NO,    KC_P7,    KC_P8,    KC_P9,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_P7,    KC_P8,    KC_P9,    KC_NO,    KC_NO,    TO(0), \
+		KC_NO,    KC_P4,    KC_P5,    KC_P6,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_P4,    KC_P5,    KC_P6,    KC_NO,              KC_PENT, \
+		KC_NO,    KC_P1,    KC_P2,    KC_P3,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_P1,    KC_P2,    KC_P3,    KC_NO,              KC_NO, \
+		KC_NO,    KC_P0,    KC_PDOT,                      KC_ENT,                       KC_P0,    KC_PDOT,  KC_NO,    KC_NO,              KC_NO),
+	[_LAYER4] = LAYOUT( /* WIN */
 		KC_GESC,  KC_1,     KC_2,     KC_3,  KC_4,  KC_5,  KC_6,    KC_7,  KC_8,    KC_9,     KC_0,            KC_MINS,  KC_EQL,  KC_BSPC, \
 		KC_TAB,   KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,    KC_U,  KC_I,    KC_O,     KC_P,            KC_LBRC,  KC_RBRC, KC_BSLASH, \
 		CTL_T(KC_CAPS),  KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,    KC_J,  KC_K,    KC_L,     KC_SCLN,         KC_QUOT,           KC_ENT, \
 		KC_LSFT,  KC_Z,     KC_X,     KC_C,  KC_V,  KC_B,  KC_N,    KC_M,  KC_COMM, KC_DOT,   RSFT_T(KC_SLSH), KC_UP,             LT(2, KC_DEL), \
 		KC_LCTL,  KC_LGUI,  KC_LALT,                KC_SPC,                KC_RALT,  MO(5)  , KC_LEFT,         KC_DOWN,           KC_RIGHT),
 	[_LAYER5] = LAYOUT( /* FN */
-		TO(3),    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL , \
+		KC_BRMD,  KC_BRMU, LCTL(KC_UP), LSFT(KC_F12),     KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL , \
```

Comment:
I recommend using `KC_BRIU` and `KC_BRID` here, as they will work on all OSes, whereas `KC_BRMU` and `KC_BRMD` are just aliases for Pause and Scroll Lock.

Reply:
Done, thank you :)